coalib/bearlib/aspects/Smell.py

```diff
@@ -294,24 +510,52 @@ class docs:
 @Smell.subaspect
 class Complexity:
     """
-    This aspect checks on the cyclomatic complexity of your code.
+    Complexity of a code based on different complexity metrics.
     """
     class docs:
         example = """
-        for (i=0; i<n; ++i){
```

Comment:
good example for complex code are very long adding one here wouldn't be such a good idea, that is why i gave a link to one